test/e2e/serial/tests/resource_accounting.go

```diff
@@ -284,65 +291,80 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload resourc
 			var ok bool
 			targetNodeName, ok = nrtCandidateNames.PopAny()
 			Expect(ok).To(BeTrue(), "cannot select a node among %#v", nrtCandidateNames.List())
-			By(fmt.Sprintf("selecting node to schedule the pod: %q", targetNodeName))
+			By(fmt.Sprintf("selecting node to schedule the test pod: %q", targetNodeName))
 
 			var err error
 			targetNodeNRTInitial, err = e2enrt.FindFromList(nrtCandidates, targetNodeName)
 			Expect(err).NotTo(HaveOccurred())
 
-			//get maximum zone CPU and Memory to be sure it wont fit on any zone
-			maxResources := corev1.ResourceList{
-				corev1.ResourceCPU:    maxResourceType(*targetNodeNRTInitial, corev1.ResourceCPU),
-				corev1.ResourceMemory: maxResourceType(*targetNodeNRTInitial, corev1.ResourceMemory),
-			}
+			//calculate base load on the target node
+			baseload, err := nodes.GetLoad(fxt.K8sClient, targetNodeName)
+			ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing node load info for %q", targetNodeName)
+			By(fmt.Sprintf("computed base load: %s", baseload))
 
-			// add a min ammount of resources just in case all the zones
-			// are equal so the pod wont fit on any of them
-			excessRes := corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("100m"),
-				corev1.ResourceMemory: resource.MustParse("100M"),
+			//get maximum node CPU and Memory
+			maxResources := corev1.ResourceList{
+				corev1.ResourceCPU:    availableResourceType(*targetNodeNRTInitial, corev1.ResourceCPU),
+				corev1.ResourceMemory: availableResourceType(*targetNodeNRTInitial, corev1.ResourceMemory),
```

Comment:
can we use node allocatable? we should document why/why not

Reply:
node total allocatable might be good in case we certainly know that there is no pods scheduled on the nodes. I wouldn't prefer setting it to allocatable just to avoid noisy pods leftovers.